docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -142,26 +151,46 @@ public class TestPluginLoader implements PluginLoader {
 Currently, you are able to add two different library types, ``JarLibrary``, and ``MavenLibraryResolver``.
 
 
-## Differences 
+## Differences
 
 ### Bukkit Serialization System
-Paper plugins do not support the serialization system (``org.bukkit.configuration.serialization``) that Bukkit uses. Your classes will not be able to be
-serialized, vise versa. It is highly recommended to not use this system with Paper plugins.
+Paper plugins still support the serialization system (``org.bukkit.configuration.serialization``) that Bukkit uses. However, custom classes will not be
+automatically registered for serialization. In order to use `ConfigurationSection#getObject`, 
+you must call `ConfigurationSerialization.registerClass(Class)` before you attempt to fetch objects from configurations.
 
 ### Classloading Isolation
-Paper plugins have isolated classloaders, meaning that relocating dependencies will not be necessary. Paper plugins are not able to access each other unless given explicit access
-by depending on another plugin, etc. This prevents plugins from accidentally accessing your dependencies, and in general helps ensure that plugins are only able to access your plugin
-if they explicitly say that they're depending on it.
+Paper plugins have isolated classloaders, meaning that relocating dependencies will not be necessary.
+Paper plugins are not able to access each other unless given explicit access by depending on another plugin, etc.
```

Comment:
This is confusing, since it's not a two-way relationship but just one-way by declaring it in your dependency section, and remove the "etc."